### PR TITLE
feature/Create script for generate release drafter workflow for each repo in axonivy-market org

### PR DESCRIPTION
**Pull request by ndkhanh-axonivy**
…axonivy-market org

### CODE REVIEW

File: createPrsToGenerateReleaseDrafterWorkflow.sh [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+# Using https://cli.github.com/
+
+org="axonivy-market"
+
+ignored_repos=(
+  "market-up2date-keeper"
+  "market.axonivy.com"
+  "market-monitor"
+  "market"
+  "demo-projects"
```
Comment (nqhoan-axonivy) [inline createPrsToGenerateReleaseDrafterWorkflow.sh:16]: Portal team also working on this stuff? do we need to check with them to ignore portal also? then portal will handle it?

File: createPrsToGenerateReleaseDrafterWorkflow.sh [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, reopened, synchronize]
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: axonivy-market/github-workflows/.github/workflows/release-drafter.yml@v4'
```
Comment (nqhoan-axonivy) [inline createPrsToGenerateReleaseDrafterWorkflow.sh:32]: not sure v4 it works for you

File: createPrsToGenerateReleaseDrafterWorkflow.sh [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  # Commit the changes
+  git add .github/workflows/release-drafter.yml
+  git commit -m "Add release-drafter workflow"
+
+  # Push the branch
+  git push origin "$branch_name"
+
+  # Create a pull request
+  pr_title="MARP-620 Add release-drafter workflow"
+  gh pr create --title "$pr_title" --body "This PR adds the Release Drafter workflow to the repository." --base master --head "$branch_name"
```
Comment (nqhoan-axonivy) [inline createPrsToGenerateReleaseDrafterWorkflow.sh:88]: should we add a -l, --label <name> to skip this pr in the release note?

**Review by nqhoan-axonivy — CHANGES_REQUESTED** [review]

**Review by ivy-rew — COMMENTED** [review]
I think this is the wrong repo; the release-drafter is a pipeline yes ... but it doesn't help in any way to supply the metrics that 'monitor' needs. 
So I'd either invent a new repo that you use to run bulk operations on the market or you use the existing one that was designed just for this: https://github.com/axonivy-market/market-up2date-keeper

File: createPrsToGenerateReleaseDrafterWorkflow.sh (hunk 3 above)
Comment (ndkhanh-axonivy) [reply createPrsToGenerateReleaseDrafterWorkflow.sh:88]: Yes, I will handle it

File: createPrsToGenerateReleaseDrafterWorkflow.sh (hunk 1 above)
Comment (ndkhanh-axonivy) [reply createPrsToGenerateReleaseDrafterWorkflow.sh:16]: After checking with Portal, I see they agree using workflow from the github-workflows

**Pull request closed without merging**